pkg/apis/azuredisk/v1alpha1/types.go

```diff
@@ -69,11 +69,13 @@ type AzVolumeAttachment struct {
 // AzVolumeAttachmentSpec is the spec for a AzVolumeAttachment resource
 type AzVolumeAttachmentSpec struct {
 	UnderlyingVolume string `json:"underlyingVolume"`
+	AzDriverNodeName string `json:"azDriverNodeName"`
```

Comment:
Is this different than the K8s node name? If not, we should just use `NodeName` here.

Reply:
I am actually not sure how azDriverNode is going to implement this. @abhisheksinghbaghel could we use NodeName here?

Reply:
The name appears to be the same as `NodeID`. See [`RegisterAzDriverNode`](https://github.com/abhisheksinghbaghel/azuredisk-csi-driver/blob/ea91490f195d66477be3a1869c7b8b1728f9cec6/pkg/azuredisk/azuredisk_v2.go#L336).

Reply:
Actually, @sunpa93 has added this as [`NodeName`](https://github.com/abhisheksinghbaghel/azuredisk-csi-driver/blob/e2bcfa80eed284d40596b034b41dc4651c4937e6/pkg/apis/azuredisk/v1alpha1/types.go#L79) in #17 .